### ISSUE TITLE
Remove unused page

### ISSUE THREAD

**Issue by discospiff** [sourav786/XML-Project#8]
It looks like this page is not used... remove it.

https://github.com/sourav786/XML-Project/blob/294e2e83544b0d54308c0aa924f24a29b2ec2a09/Pages/htmlpage.html#L1

**Comment by sourav786** [comment]
Yes its a miss from our side.. I will keep this comments in mind for this project as well as future projects too...

Thanks you so much Professor providing an understanding of skeleton structure of the project.
